templates/errors/404.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'main/base.html' %}
+{% block title %} 404 - Page not found! {% endblock %}
+{% block beta_error %}{% endblock %}
+{% block content %}
+    <div class="uk-background-muted">
+        <div class="uk-section">
+            <div class="uk-text-center"><code style="font-size: 170%;">AttributeError: Page not located!</code></div>
```

Comment:
Are you sure there's no UIKit class that can make the text the size that you want it?

Reply:
Upon looking at https://getuikit.com/docs/text I think `.uk-text-large` should do the job.